cdc/owner/changefeed.go

```diff
@@ -501,6 +502,17 @@ LOOP:
 	cancelCtx, cancel := cdcContext.WithCancel(ctx)
 	c.cancel = cancel
 
+	sourceID, err := pdutil.GetSourceID(ctx, c.upstream.PDClient)
```

Comment:
it is not necessary to get sourced if  the BDR is disabled

Reply:
yes, we always need it.